### DISCUSSION TITLE
Document `site.Params.mainSections` usage

### DISCUSSION THREAD

**Discussion by cgalo5758** [adityatelange/hugo-PaperMod#1081] [Ideas]
I was pulling my hairs out trying to figure out why my homepage would only show content from only one section. I found out that the theme filters sections for the homepage using the [mainSections](https://gohugo.io/functions/where/#mainsections) parameter; This means that the homepage will by default show only the section with the most content.

**Comment by yriveiro** [comment]
I spent about a day on this as well, I would like to have a section for posts and another for books and display the book section on the main menu, but it seems that I can't have both :/

**Reply by cgalo5758** [reply]
This has already been fixed:

https://github.com/adityatelange/hugo-PaperMod/issues/1082
